generateHtmlTemplates.js

```diff
@@ -0,0 +1,103 @@
+const ejs = require("ejs");
+const fs = require("fs-extra");
+const path = require("path");
+
+// funzione per ricavare le traduzioni dalla cartella i18n corrispondente
+async function loadTranslations(language, template) {
+  const filePath = path.join(
+    __dirname,
+    "resources/templates/" + template,
+    "/i18n",
+    `${language}.json`
+  );
+  const result = await fs.readJson(filePath);
+
+  const itaText = path.join(
+    __dirname,
+    "resources/templates/" + template,
+    "/i18n",
+    `it.json`
+  );
+  const itaResult = await fs.readJson(itaText);
+
+  const mergedJSON = Object.assign({}, result, itaResult);
+  return mergedJSON;
+}
+
+// generazione template BILINGUA
+async function generateHtmlTemplates() {
+  const languages = ["it", "de", "fr", "sl"]; // Elenca le lingue supportate
+  const templates = [
+    "NotificationAAR_RADDalt_bilingual",
+    "NotificationAAR_bilingual",
+    "NotificationReceivedLegalFact",
+    "PecDeliveryWorkflowLegalFact",
+    "NotificationViewedLegalFact",
+    "LegalFactMalfuction",
+    "AnalogDeliveryWorkflowFailureLegalFact",
+    "NotificationCancelledLegalFact",
+    "NotificationAARForEMAIL",
+    "NotificationAARForPEC"
+  ];
+
+  for (const template of templates) {
+    const templatePath = path.join(
+      __dirname,
+      "resources/templates",
+      template,
+      "index.html"
+    );
+    const templateContent = await fs.readFile(templatePath, "utf8");
+
+    for (const language of languages) {
+      const fileName = language === "it" ? template : `${template}_${language}`;
+      const templateLang =
+        language === "it"
+          ? templateContent.replaceAll(
+              /<[^>]*data-hide-on-it="true"[^>]*>[\s\S]*?<\/[^>]*>/g,
```

Comment:
This not work on nested elements. Try on regex101 and use and example string like this

```
<div>
 <div-to-remove data-hide-on-it="true">
  <p>
   This a text for testing
  </p>
 </div-to-remove>
</div>
```

you will see that the regexp will stop at p closing tag. This happen because the regular expression stops to the first closing tag regardless of whether it is the same as opening with data-hide-on-it.

Maybe we can manage the task in a simple way:
- or define a custom tag (i.e. `<start-lang></end-lang>`). this tag must be always removed with the difference that in italian must me removed also the content
- or define a comment (i.e. `<!-- START lang --> and <!-- END lang -->`)